rust/integration-tests/src/sdk_tests/test_cli_flag_util.rs

```diff
@@ -0,0 +1,43 @@
+#[allow(dead_code)]
+#[derive(Debug, Clone)]
+pub struct TestArgs {
```

Comment:
I think we can use `clap` crate to help parse the CLI args so you don't have to parse the args manually. We use this for the processors to parse the config file.

Reply:
yeah i tried but it seems like it doesn't support args parsing with `cargo test` see https://github.com/rust-lang/cargo/issues/8916